--- a/src/panfrost/model/pan_model.c	2026-02-25 16:54:24.000000000 +0100
+++ b/src/panfrost/model/pan_model.c	2026-09-24 17:33:13.890232684 +0200
@@ -85,6 +85,9 @@
                                               MODEL_RATES(2, 4,  32)),
    VALHALL_MODEL(0x9003, 0, "G57",    "TNAx", MODEL_ANISO(ALL),  MODEL_TB_SIZES(16384,  8192),
                                               MODEL_RATES(2, 4,  32)),
+   VALHALL_MODEL(0x9202, 0, "G78",    "TBOx", MODEL_ANISO(ALL),  MODEL_TB_SIZES(16384,  8192),
+                                              MODEL_RATES(2, 4,  32),
+                                              MODEL_QUIRKS( .no_crc = true )),
    VALHALL_MODEL(0xa807, 0, "G610",   "TVIx", MODEL_ANISO(ALL),  MODEL_TB_SIZES(32768, 16384),
                                               MODEL_RATES(4, 8,  64)),
    VALHALL_MODEL(0xac04, 0, "G310v1",   "TVAx", MODEL_ANISO(ALL),  MODEL_TB_SIZES(16384,  8192),
--- a/src/panfrost/model/pan_model.h	2026-02-25 16:54:24.000000000 +0100
+++ b/src/panfrost/model/pan_model.h	2026-09-24 17:33:13.890354465 +0200
@@ -75,6 +75,7 @@
        */
       bool no_hierarchical_tiling;
       bool max_4x_msaa;
+      bool no_crc;
    } quirks;
 };
 
--- a/src/gallium/drivers/panfrost/pan_resource.c	2026-02-25 16:54:24.000000000 +0100
+++ b/src/gallium/drivers/panfrost/pan_resource.c	2026-09-24 17:33:13.897886263 +0200
@@ -813,6 +813,9 @@
 panfrost_should_checksum(const struct panfrost_device *dev,
                          const struct panfrost_resource *pres)
 {
+   if (dev->model->quirks.no_crc)
+      return false;
+
    /* When checksumming is enabled, the tile data must fit in the
     * size of the writeback buffer, so don't checksum formats
     * that use too much space. */
